web/src/components/vis/PcaPage/LoadingsPlot.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+import LoadingsPlot from '@/components/vis/LoadingsPlot.vue';
+import VisTile from '@/components/vis/VisTile.vue';
+import plotData from '@/components/vis/mixins/plotData';
+
+export default {
+  components: {
+    LoadingsPlot,
+    VisTile,
+  },
+
+  mixins: [
+    plotData('loadings'),
+  ],
+
+  props: {
+    id: {
+      type: String,
+      required: true,
+    },
+
+    pcX: {
+      type: Number,
+      validator: prop => Number.isInteger(prop),
+      required: true,
+    },
+
+    pcY: {
+      type: Number,
+      validator: prop => Number.isInteger(prop),
+      required: true,
+    },
+
+    showCrosshairs: {
+      type: Boolean,
+      required: true,
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+vis-tile(title="PCA Loadings Plot", :loading="plot.loading", svg-download, v-if="plot")
```

Comment:
what happened to the help texts?

Reply:
Added help panels back in via 51f80a4 and 56a9d41.